backend/corpora/common/entities/dataset.py

```diff
@@ -124,6 +124,62 @@ def get_asset(self, asset_uuid) -> typing.Union[DatasetAsset, None]:
         asset = [asset for asset in self.artifacts if asset.id == asset_uuid]
         return None if not asset else DatasetAsset(asset[0])
 
+    @staticmethod
+    def transform_sex_for_schema_2_0_0(dataset):
+        # If schema_version is 1.1.0, convert sex to the new API format
+        if "sex" in dataset and dataset.get("schema_version") != "2.0.0":
+            dataset["sex"] = [{"label": s, "sex_ontology_term_id": "unknown"} for s in dataset["sex"]]
+
+    @staticmethod
+    def transform_organism_for_schema_2_0_0(dataset):
+        # If organism is an object (version 1.1.0), wrap it into an array to be 2.0.0 compliant
+        if "organism" in dataset and dataset.get("schema_version") != "2.0.0":
+            dataset["organism"] = [dataset["organism"]]
+
+    @classmethod
+    def list_for_index(cls, session) -> typing.List[typing.Dict]:
+        """
+        Return a list of all the datasets. For efficiency reasons, this only returns the fields
+        inside the `dataset` table and doesn't include relationships.
+        """
+
+        attrs = [
+            DbDataset.id,
+            DbDataset.name,
+            DbDataset.collection_id,
+            DbDataset.tissue,
+            DbDataset.disease,
+            DbDataset.assay,
+            DbDataset.organism,
+            DbDataset.cell_count,
+            DbDataset.cell_type,
+            DbDataset.sex,
+            DbDataset.ethnicity,
+            DbDataset.development_stage,
+            DbDataset.schema_version,
+            DbDataset.tombstone,
```

Comment:
tombstone field needed if we're always filtering out tombstoned (below)?  Should also document in method comment and API spec that tombstoned are excluded.

Reply:
Actually, I think it's probably better to move the tombstone filtering to a `where` condition, as it will improve the transfer time between the database and the server. Therefore this can be removed.